tests/conftest.py

```diff
@@ -23,6 +23,12 @@ def pytest_addoption(parser):
                      action="store",
                      help="Redis connection string,"
                           " defaults to `%(default)s`")
+    parser.addoption('--docker', default=False, action="store_true",
```

Comment:
I'm using this to decide of the slave host is "slave" (docker environment) or the same host as the master -- based on the --redis-url parameter.

This doesn't quite feel right but making it more correct (according to my own view) would require more pytest options, like --slave-url.